packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -249,12 +248,8 @@ export const capturePhaseEvents: Set<DOMTopLevelEventType> = new Set([
 ]);
 
 if (!enableFormEventDelegation) {
-  capturePhaseEvents.add(TOP_SUBMIT);
-  capturePhaseEvents.add(TOP_RESET);
-}
-
-if (enableCreateEventHandleAPI) {
-  capturePhaseEvents.add(TOP_AFTER_BLUR);
```

Comment:
Why does it not need special handling anymore?

Reply:
It wasn't ever needed in the capture phase after it was refactored. I must have missed this line.